R/zchunk_LB134.Diet_Rfao.R

```diff
@@ -29,7 +29,7 @@ module_aglu_LB134.Diet_Rfao <- function(command, ...) {
              "L100.FAO_ag_Food_t",
              "L100.FAO_an_Food_t",
              "L101.ag_Food_Pcal_R_C_Y",
-             FILE = "temp-data-inject/L105.an_Food_Pcal_R_C_Y",
```

Comment:
Did you remove this file from `temp-data-inject/`? Did I miss the removal in this PR?